pyxem/tests/signals/test_diffraction_vectors.py

```diff
@@ -19,11 +19,30 @@
 import pytest
 import numpy as np
 from sklearn.cluster import DBSCAN
+from skimage.draw import disk
+from skimage.morphology import disk as disk2
```

Comment:
What is going on here?

Reply:
The skimage namespace is a little crowded.  The skimage.draw.disk returns xx,yy coordinates whole the skim age.morphology returns a 2D disk kernel.

Reply:
weird